### PR TITLE
BAQE-1693 - kogito-examples - unpack-build-maven-plugin replaced by m…

### PR DESCRIPTION
**Pull request by hellowdan**
…aven-dependency-plugin

### CODE REVIEW

File: run-tests-with-build-kogito-examples/pom.xml [hunk 1]
```diff
@@ -12,47 +12,38 @@
   <artifactId>run-tests-with-build-kogito-examples</artifactId>
 
   <properties>
-    <sources.url>${sources.url}</sources.url>
+    <!-- integration-tests-quarkus-decisions tests depends on src/main/java classes -->
```
Comment (jomarko) [inline run-tests-with-build-kogito-examples/pom.xml:15]: Could we please update the comment?

`integration-tests-quarkus-decisions` refers to submodule of `kogito-runtimes`.

In the context of `kogito-examples`, this is needed for example due to submodule `dmn-listener-quarkus`, as there are some java classes under `src/main/java` used during test runtime.

File: run-tests-with-build-kogito-examples/pom.xml [hunk 2]
```diff
@@ -61,42 +52,60 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-invoker-plugin</artifactId>
         <configuration>
+          <!-- kogito-runtimes tests were not designed to run in parallel -->
```
Comment (jomarko) [inline run-tests-with-build-kogito-examples/pom.xml:55]: I think also here we should refer `kogito-examples` rather than `kogito-runtimes`

**Review by jomarko — COMMENTED** [review]
@hellowdan thank you, two minor comments. Otherwise look (and locally run) well.

**Review by jomarko — APPROVED** [review]
@hellowdan thank you for updates

File: run-tests-with-build-kogito-examples/pom.xml [hunk 3]
```diff
@@ -12,47 +12,38 @@
   <artifactId>run-tests-with-build-kogito-examples</artifactId>
 
   <properties>
-    <sources.url>${sources.url}</sources.url>
+    <!-- Tthere are some java classes under src/main/java used during test runtime (dmn-listener-quarkus for example) -->
```
Comment (jstastny-cz) [inline run-tests-with-build-kogito-examples/pom.xml:15]:
```suggestion
    <!-- There are some java classes under src/main/java used during test runtime (dmn-listener-quarkus for example) -->
```

**Review by jstastny-cz — APPROVED** [review]
One type, otherwise looks fine.

**Pull request merged**